src/transformers/benchmark/benchmark.py

```diff
@@ -58,15 +68,25 @@ def train(self, model_name, batch_size, sequence_length, trace_memory=False):
                 vocab_size, (batch_size, sequence_length), dtype=torch.long, device=self.args.device
             )
 
+            if self.args.torchscript:
+                raise NotImplementedError("Training for torchscript is currently not implemented")
```

Comment:
@LysandreJik I think to implement torchscript for training, I need to pass all tensors until labels. I don't think we always have the same number of tensors before `labels` in each model's forward(). So not sure what to do here...Could do something ugly with `if else statements depending on the model class`. What do you think?